app/models/ability.rb

```diff
@@ -13,6 +13,7 @@ def initialize(user)
     else
       external_user_permissions(user)
     end
+    can :create, Participant # TODO: define permissions for participants
```

Comment:
i will come up with a suggestion:

```rb
  def external_user_permissions(user)
    ... 
    can :manage, Participant, unit: { lagerleiter: { pbs_id: user.pbs_id } }
    can :manage, Participant, unit: { al: { pbs_id: user.pbs_id } }
  end

  def midata_user_permissions(user)
      can :read, Participant, unit: { al: { pbs_id: user.pbs_id } }
      can :read, Participant, unit: { lagerleiter: { pbs_id: user.pbs_id } }
  end
```

I think it would be worth to test. i would be glad to help.